Source/LogWriter.swift

```diff
@@ -112,24 +149,35 @@ open class OSLogWriter: LogMessageWriter {
     /// - Parameters:
     ///   - message:   The original message to write to the console.
     ///   - logLevel:  The log level associated with the message.
-    ///   - modifiers: The modifier objects to run over the message before writing to the console.
-    open func writeMessage(_ message: String, logLevel: LogLevel, modifiers: [LogMessageModifier]?) {
-        var message = message
-        modifiers?.forEach { message = $0.modifyMessage(message, with: logLevel) }
+    open func writeMessage(_ message: String, logLevel: LogLevel) {
+        let message = modifyMessage(message, logLevel: logLevel)
+        let type = logType(forLogLevel: logLevel)
+
+        os_log("%@", log: log, type: type, message)
+    }
 
-        let logType: OSLogType
+    /// Writes the breadrumb to the `OSLog` using the `os_log` function.
```

Comment:
`breadcrumb`? Probably worth searching through the codebase for this to see if there are any other leftovers.